core/src/main/scala/water/support/ModelSerializationSupport.scala

```diff
@@ -61,6 +63,18 @@ trait ModelSerializationSupport {
     destination
   }
 
+  def exportMOJOModelToHDFS(model: Model[_, _, _], destination: URI, sc: SparkContext): URI = {
```

Comment:
I would delete this method and integrate this with `exportMojoModel` method to have clean API.

The `exportMojoModel` can check the uri schema and execute hdfs specific code, otherwise run as normal

Reply:
thanks for the feedback, that makes sense. I will update accordingly